nimbus/helm/templates/nimbus-deployment.yaml

```diff
@@ -0,0 +1,95 @@
+{{- range $key, $nimbusClient := .Values.nimbus.clients }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $nimbusClient.name }}
+  namespace: nimbus
```

Comment:
For other clients, namespace is also a variable in the helm chart. Are we going to change the behavior?